Quickstart/azure-search-quickstart/azure-search-quickstart.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
The dashed naming convention is highly unusual for VS project and solution files. Normally we use PascalCase.

Reply:
renamed to AzureSearchQuickstart